src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -138978,18 +139418,30 @@ static T F1<T>(T x1)
 }";
             var comp = CreateCompilation(source, parseOptions: TestOptions.Regular9);
             comp.VerifyDiagnostics(
+                // (7,14): warning CS8600: Converting null literal or possible null value to non-nullable type.
```

Comment:
It occurs to me that this message may not be quite right. `T` is a *possibly* non-nullable type. It is not known to be non-nullable. However, this may not end up being very disruptive/confusing to users in practice.